frappe/core/doctype/server_script/server_script_dashboard.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+
+from frappe import _
+
+
+def get_data():
+	return {
+		"fieldname": "server_script",
+		"transactions": [
+			{"label": _("Scheduled Jobs"), "items": ["Scheduled Job Type"]},
```

Comment:
This can now be configured from Document Links table in the doctype form. No need for a python file.